dotnet/src/dotnetcore/Providers/Messaging/GXMessageBroker/MessageBrokerProvider.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using GeneXus.Attributes;
+using GeneXus.Encryption;
+using GeneXus.Services;
+using GeneXus.Utils;
+using GxClasses.Helpers;
+using log4net;
+
+namespace GeneXus.Messaging.Common
+{
+	[GXApi]
+	public class MessageBrokerProvider : MessageQueue
+	{
+		static readonly ILog logger = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+		private static GXService providerService;
+		public MessageBrokerProvider()
+		{
+
+		}
+
+		public MessageQueue Connect(string providerTypeName, GXProperties properties, out GXBaseCollection<SdtMessages_Message> errorMessages, out bool success)
+		{
+			errorMessages = new GXBaseCollection<SdtMessages_Message>();
+			MessageQueue messageQueue = new MessageQueue();
+			if (string.IsNullOrEmpty(providerTypeName))
+			{
+				GXUtil.ErrorToMessages("GXMessageBroker1000", "Message Broker provider cannot be empty", errorMessages);
```

Comment:
"'GXMessageBroker1000". Why the "1000"?

Reply:
You suggest starting with code 1? or what do you suggest? thanks

Reply:
I removed the numbers.